### PR TITLE
fix(droid): Keyboard overlapping controls

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #
fixes https://github.com/unoplatform/nventive-private/issues/83

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
Android 11 keyboard is overlapping controls

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/LayoutProvider.Android.cs [hunk 1]
```diff
@@ -53,8 +53,8 @@ internal void Start(View view)
 		{
 			if (view?.WindowToken != null)
 			{
-				_adjustNothingLayoutProvider.Start(view);
 				_adjustResizeLayoutProvider.Start(view);
+				_adjustNothingLayoutProvider.Start(view);
```
Comment (jeromelaban) [inline src/Uno.UI/LayoutProvider.Android.cs:57]: We should also call `Stop` when the view changes.

Comment (ajpinedam) [reply src/Uno.UI/LayoutProvider.Android.cs:60]: Updated as we talked

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/3.7

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/3.1

**Pull request merged**